Sources/CLI/System/Kernel/KernelSet.swift

```diff
@@ -89,7 +90,7 @@ extension Application {
             try await Self.downloadAndInstallWithProgressBar(tarRemoteURL: remoteURL.absoluteString, kernelFilePath: binaryPath, platform: platform, force: force)
         }
 
-        private func getSystemPlatform() throws -> SystemPlatform {
+        func getSystemPlatform() throws -> SystemPlatform {
```

Comment:
Same here, if you don't need them now, let's expose them when and if we come to need them.